### PR TITLE
Fix "PHP command not found" error

### PR DESCRIPTION
**Pull request by staabm**
this fixes the php error for me. maybe it works for @samsonasik now?

I got a new error now

grafik

### CODE REVIEW

File: src/DemoRunner.php [hunk 1]
```diff
@@ -68,15 +68,16 @@ private function processFilesContents(string $fileContent, string $configContent
     {
         $identifier = Random::generate(20);
 
-        $analyzedFilePath = $this->demoDir . '/' . $identifier . '/' . self::ANALYZED_FILE_NAME;
-        $configPath = $this->demoDir . '/' . $identifier . '/' . self::CONFIG_NAME;
+        $analyzedFilePath = $this->demoDir . DIRECTORY_SEPARATOR . $identifier . DIRECTORY_SEPARATOR . self::ANALYZED_FILE_NAME;
```
Comment (staabm) [inline src/DemoRunner.php:71]: needed for windows compat (as I am using windows on my work PC)

File: src/DemoRunner.php [hunk 2]
```diff
@@ -68,15 +68,16 @@ private function processFilesContents(string $fileContent, string $configContent
...
+        $configPath = $this->demoDir . DIRECTORY_SEPARATOR . $identifier . DIRECTORY_SEPARATOR. self::CONFIG_NAME;
 
         $this->filesystem->dumpFile($analyzedFilePath, $fileContent);
         $this->filesystem->dumpFile($configPath, $configContent);
 
         $temporaryFilePaths = [$analyzedFilePath, $configPath];
 
         $process = new Process([
+            PHP_BINARY ,
```
Comment (staabm) [inline src/DemoRunner.php:80]: pass the PHP command thru, makes "command not found" go away

**Review by samsonasik — APPROVED** [review]
that seems works now 👍

**Comment by samsonasik** [comment]
Thank you @staabm , let's give it a try.

**Comment by TomasVotruba** [comment]
Great job @staabm :clap:

**Comment by samsonasik** [comment]
@staabm I committed from server, check `prod` env should be a tweak

https://github.com/rectorphp/getrector-com/commit/7179151c827d77274c88eb09593dde2339328688

**Pull request merged**